packages/astro/src/stores/external.ts

```diff
@@ -4,6 +4,16 @@ import { computed, onMount, type Store } from 'nanostores';
 
 import { $clerk, $csrState, $initialState } from './internal';
 
+/**
+ * A client side store that returns the loaded state of clerk-js.
+ *
+ * @example
+ * A simple example:
+ *
+ * $isLoadedStore.subscribe((authloaded => console.log(loaded))
+ */
+export const $isLoadedStore = computed([$csrState], state => state.isLoaded);
```

Comment:
Exporting the Clerk loaded status here as our `$clerkStore` atom is not reliable atm to check for loading status

Reply:
What's the reliability issue with `$clerkStore` ?

Reply:
Say you want to subscribe to it to get the `loaded` status, it will always end up in the `false` value and all other properties like `user`, etc. are undefined.

```ts
import { $clerkStore } from '@clerk/astro/client'

$clerkStore.subscribe((clerk) => {
  console.log('clerk loaded', clerk?.loaded)
  console.log('clerk user', clerk?.user)
})
```

<img width="293" alt="Screenshot 2024-08-21 at 9 11 00 AM" src="https://github.com/user-attachments/assets/7ef313a5-3ae5-44c1-a751-c632fe626ec9">

This is something that I've been trying to explore on why it's happening. Also while we're here, when subscribing to other stores we export, it emits the initial value even after Clerk is loaded:

```ts
import { $authStore } from '@clerk/astro/client'

$authStore.subscribe((state) => {
  console.log('$authStore', state)
})
```

<img width="648" alt="Screenshot 2024-08-21 at 9 14 08 AM" src="https://github.com/user-attachments/assets/367400e4-6b9d-452b-8f49-8e1401954de8">

Reply:
the issue with loaded, would it work if we were to do this ?

``ts
clerkJSInstance.load().then(() => $clerk.set(clerkJSInstance)) 
```

Reply:
I tried that one @panteliselef but didn't work as expected. `loaded` is still `false` and other properties are still undefined. I even try to set the instance again every update inside the `addListener` with no luck.

Will do a separate investigation on this one 👍🏼  But I can confirm that this not only happens in our app. I get this behavior in svelte-clerk too